go.mod

```diff
@@ -6,10 +6,10 @@ require (
 	github.com/richardlehane/mscfb v1.0.3
 	github.com/richardlehane/webarchive v1.0.0
 	github.com/richardlehane/xmldetect v1.0.2
-	github.com/ross-spencer/spargo v0.0.0-20200323024642-38971d4365a7
-	golang.org/x/image v0.0.0-20200922025426-e59bae62ef32
-	golang.org/x/sys v0.0.0-20200922070232-aee5d888a860
-	golang.org/x/text v0.3.3 // indirect
+	github.com/ross-spencer/wikiprov v0.0.0-20210125004524-30448c098502
```

Comment:
Not sure I know enough about the mod file to understand why x/image and x/sys are updated here. Wikiprov is expected.